### PR TITLE
docs: Deployments

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Adds individual pages for the different Honeycomb deployments
- Updates the CI page with the Firebase workflows

### CODE REVIEW

File: docs/ci.mdx [hunk 1]
```diff
@@ -17,26 +17,35 @@ CI/CD refers to Continuous Integration and either Continuous Delivery or Continu
...
+- `Clinic`: The app expects event code triggers and photodiode spots.
 
-More specifically, the following workflows are included:
+:::tip
+Event code triggers and photodiode spots can only be used on local applications! They will not appear when Honeycomb is deployed on the web.
+:::
 
-- `build.yaml`: Every time an Pull Request (PR) is opened, or a push is made to the `main` branch, the software is built and tests are run for all platforms and `home` and `clinic` settings. This workflow does not build and upload desktop installers
-- `package.yaml`: Create installers for any/all platforms for the `home` and `clinic` setting on demand <sup>1</sup>. The installers/executables are uploaded as artifacts and are available for download from the GitHub Actions tab. This also builds a PsiTurk version when linux or all operating systems are selected.
+- `build.yaml`: Every time an Pull Request (PR) is opened, or a push is made to the `main` branch, the software is built and tests are run for all platforms with `home` and `clinic` settings, as well as `video on` and `video of`. This workflow does not upload desktop installers.
```
Comment (anna-murphy) [inline docs/ci.mdx]: Is this supposed to be "video off"?

```suggestion
- `build.yaml`: Every time an Pull Request (PR) is opened, or a push is made to the `main` branch, the software is built and tests are run for all platforms with `home` and `clinic` settings, as well as `video on` and `video off`. This workflow does not upload desktop installers.
```

File: docs/deployments/gh_pages.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+id: gh_pages
+slug: /gh_pages
+title: GitHub Pages
+description: Guide for deploying a task to GitHub Pages
+---
+
+The `.github/workflows/release.yml` workflow will automatically deploy your task to GitHub pages if you have it enabled for your repository. It uploads the built website to the `gh-branch` branch. You can then access your task at `<github user>.github.io/<repository>`.
```
Comment (anna-murphy) [inline docs/deployments/gh_pages.mdx]: Is `gh-branch` supposed to be `gh-pages`? (You refer to it as `gh-pages` in a later paragraph, and also that's the title of the file.)

```suggestion
The `.github/workflows/release.yml` workflow will automatically deploy your task to GitHub pages if you have it enabled for your repository. It uploads the built website to the `gh-pages` branch. You can then access your task at `<github user>.github.io/<repository>`.

```

**Review by anna-murphy — APPROVED** [review]
Looks good!  There were a few small wording things / typos I pointed out but other than that I think you're okay!

File: docs/deployments/gh_pages.mdx (hunk 2 above)
Comment (RobertGemmaJr) [reply docs/deployments/gh_pages.mdx]: Yup! Good eye!

**Review by tdivoll — COMMENTED** [review]
For the failing test, it seems like the `./assets/<path>` is not being found. Maybe use full paths? Or try `./docs/assets/<path>`?

**Comment by RobertGemmaJr** [comment]
> For the failing test, it seems like the `./assets/<path>` is not being found. Maybe use full paths? Or try `./docs/assets/<path>`?

Relative path, and `firebase.mdx` got moved! Had to change it to `../assets.....`

**Pull request merged**